articles/defining-and-using-observables.md

```diff
@@ -281,7 +281,12 @@ In this example, `uid` is an observable by dictionary type with `type_id` of `10
   ],
   "observables": [
     {
-      "name": "resources.uid",         //Generic notation
+      "name": "cloud.account.uid",      // Basic, non-array path
+      "type_id": 35,
+      "value": "111122223333"
+    },
+    {
+      "name": "resources.uid",         //Simple array notation
```

Comment:
The `[]` seems missing here, no?

Reply:
yeah good catch, I'll add an example for that case too